src/prettytable/prettytable.py

```diff
@@ -74,6 +76,61 @@ def _get_size(text):
     return width, height
 
 
+##############################
+# ALIGNMENT VALIDATORS       #
+##############################
+
+
+def validate_align(val):
```

Comment:
Should these three validators be private/internal and not public?

And let's add type hints.

```suggestion
def _validate_align(val: str) -> None:
```